homeassistant/components/flunearyou/strings.json

```
@@ -7,12 +7,14 @@
         "data": {
           "latitude": "[%key:common::config_flow::data::latitude%]",
           "longitude": "[%key:common::config_flow::data::longitude%]"
-	}
+        }
       }
     },
-    "error": { "general_error": "There was an unknown error." },
+    "error": {
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
     "abort": {
       "already_configured": "These coordinates are already registered."
```

```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```
This is a common string as well.

---

I would rather use `[%key:common::config_flow::abort::already_configured_service%]`

---

I wonder if we should introduce a new one, `already_configured_location` ?